docs/launch_npb_tests.py

```diff
@@ -100,10 +100,12 @@
 		run = gem5Run.createFSRun(
 			'gem5/build/X86/gem5.opt',
 			'configs-npb-tests/run_npb.py',
+                       '''/results/X86/run_npb/vmlinux-5.2.3/
```

Comment:
I think it would be better to use an LTS kernel (e.g., 4.19.83, IIRC)

Reply:
Will do this in a new PR. Created an issue on Jira